PathfinderAPI/Replacements/ContentLoader.cs

```diff
@@ -711,32 +711,32 @@ static ContentLoader()
             executor.RegisterExecutor("Computer.DLCCredits", (exec, info) =>
             {
                 DLCCreditsDaemon credits;
-                string title, text = null;
-                if (info.Attributes.TryGetValue("Title", out title) || info.Attributes.TryGetValue("ButtonText", out text))
+                string title = info.Attributes.TryGetValue("Title", out string titleValue) ? titleValue : null;
+                string text = info.Attributes.TryGetValue("ButtonText", out string textValue) ? textValue : null;
```

Comment:
Why didn't you just do `info.Attributes.TryGetValue("Title", out title) && info.Attributes.TryGetValue("ButtonText", out text)`?

Reply:
Replace `||` with `|`